spec/grape/validations/validators/default_spec.rb

```diff
@@ -187,6 +187,15 @@ def app
         subject.post '/optional_hash_with_default_inner_params' do
           { foo_in_optional_hash: params[:optional_hash_with_default][:foo_in_optional_hash] }
         end
+
+        subject.params do
+          optional :phone, type: Hash do
+            optional :zone, type: String, default: '1234'
+          end
+        end
+        subject.post '/optional_hash_without_default_outer_params' do
+          { zone: params[:phone][:zone] }
+        end
```

Comment:
I think we can remove this declarition and put the expectations to [this context](https://github.com/namusyaka/grape/blob/fix-1245/spec/grape/validations/validators/default_spec.rb#L167)
```ruby
      it 'does not fail even if invalid params is provided' do
        expect { post '/optional_hash_without_default', optional_hash_without_default: '5678' }.not_to raise_error
        expect(last_response.status).to eq(400)
        expect(last_response.body).to eq({ error: 'optional_hash_without_default is invalid' }.to_json)
      end
```